### DISCUSSION TITLE
Autobuild only a single language package

### DISCUSSION THREAD

**Discussion by G-Rumpel** [leecher1337/ntvdmx64#137] [Ideas]
Hi,
Is it possible, if not already, to compile only a single language package?
I modified  autobuild-ccpu-chk.cmd to copy a single locale into release directory but this does not prevent from building all the others. If possible at all it will reduce the time for auto building significantly.
Regards

**Comment by leecher1337** [comment]
Should be possible now

**Reply by G-Rumpel** [reply]
Hi,
That one was very fast reply, thank you!
I set LANG variable in "autobuild-ccpu-fre.cmd" and now the process is a lot shorter.
But there are still messages for multiple locales as follows:

```
1>Assembling - mvdm\dos\v86\doskrnl\dos\cht\msinit.asm for X86
2>Assembling - mvdm\dos\v86\doskrnl\dos\jpn\file.asm for X86
2>Assembling - mvdm\dos\v86\doskrnl\dos\jpn\lock.asm for X86
1>Assembling - mvdm\dos\v86\doskrnl\dos\cht\origin.asm for X86
2>Assembling - mvdm\dos\v86\doskrnl\dos\jpn\msinit.asm for X86
2>Assembling - mvdm\dos\v86\doskrnl\dos\jpn\origin.asm for X86
1>Linking Executable - mvdm\dos\v86\doskrnl\dos\cht\@cht\objfre\x86\tmp.lnk for X86
2>Linking Executable - mvdm\dos\v86\doskrnl\dos\jpn\@jpn\objfre\x86\tmp.lnk for X86
1>Binplacing - mvdm\dos\v86\doskrnl\dos\cht\mvdm\dos\v86\doskrnl\dos\cht\objfre\x86\ntdos404.sys for X86
1>Binplacing - mvdm\dos\v86\doskrnl\dos\cht\mvdm\dos\v86\doskrnl\dos\cht\objfre\x86\ntdos404.map for X86
1>Binplacing - mvdm\dos\v86\doskrnl\dos\cht\mvdm\dos\v86\doskrnl\dos\cht\objfre\x86\ntdos404.sym for X86
BUILD: Compiling d:\ntvdmx64\w\minnt\base\mvdm\dos\v86\dev\himem\usa directory
2>Binplacing - mvdm\dos\v86\doskrnl\dos\jpn\mvdm\dos\v86\doskrnl\dos\jpn\objfre\x86\ntdos411.sys for X86
2>Binplacing - mvdm\dos\v86\doskrnl\dos\jpn\mvdm\dos\v86\doskrnl\dos\jpn\objfre\x86\ntdos411.map for X86
2>Binplacing - mvdm\dos\v86\doskrnl\dos\jpn\mvdm\dos\v86\doskrnl\dos\jpn\objfre\x86\ntdos411.sym for X86
```
Is this normal under the new settings?

And please, keep on doing this great job!
Regards

**Reply by leecher1337** [reply]
This is intentional, without it, there wouldn't be ntio???.sys and ntdos???.sys (i.e. ntio411.sys, ntdos411.sys), which in fact are parts of a normal NTVDM installation. The ??? is the langid. 
At least on i.e. German Windows XP, these files are present, so there may be a good reason behind it (compatibility, I guess?).

**Reply by G-Rumpel** [reply]
OK.
The case dismissed :o)
